src/lib/support/MathUtils.h

```diff
@@ -31,6 +31,40 @@ namespace nl {
 namespace Weave {
 namespace Platform {
 
+/**
+ *  @def ROUNDDOWN(a, b)
```

Comment:
These are super confusing.  `ROUNDDOWN` returns the value rounded down to the nearest multiple of rounding granularity.  `ROUNDUP` returns `ceil((double)a/(double)b)`.  These are not parallel. If `ceil`/`floor` are what we want, we should call those macros in those terms.

Reply:
Fixed ROUNDUP() macro definition.